modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/TrialFinishedNotes.kt

```diff
@@ -0,0 +1,116 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.account.onboarding.upgrade.OnboardingUpgradeHelper.plusGradientBrush
+import au.com.shiftyjelly.pocketcasts.compose.components.GradientIcon
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH50
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.images.R as IR
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+import au.com.shiftyjelly.pocketcasts.profile.R as PR
+
+private val DefaultCardShape = RoundedCornerShape(8.dp)
+
+private object NotesCardTheme {
+    val backgroundColor @Composable get() = MaterialTheme.theme.colors.primaryUi01
+    val borderColor @Composable get() = MaterialTheme.theme.colors.primaryText02
+}
+
+@Composable
+internal fun TrialFinishedNotesCard(
+    modifier: Modifier = Modifier,
+) {
+    Card(
+        shape = DefaultCardShape,
+        elevation = 8.dp,
+        backgroundColor = NotesCardTheme.backgroundColor,
+        modifier = modifier
+            .fillMaxWidth()
+            .border(width = 2.dp, color = NotesCardTheme.borderColor, shape = DefaultCardShape),
+    ) {
+        Column(modifier = Modifier.padding(bottom = 20.dp)) {
+            TrialFinishedNotesItem.entries.forEach {
+                NotesItemRow(item = it)
+            }
+        }
+    }
+}
+
+@Composable
+private fun NotesItemRow(
+    item: NotesItem,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        horizontalArrangement = Arrangement.Center,
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(top = 20.dp, start = 32.dp, end = 16.dp),
+    ) {
+        GradientIcon(
+            painter = painterResource(item.icon),
+            gradientBrush = plusGradientBrush,
+            modifier = Modifier.size(24.dp),
+        )
+
+        Spacer(Modifier.width(16.dp))
+
+        TextH50(
+            text = stringResource(item.text),
+            color = MaterialTheme.theme.colors.primaryText02,
+            fontWeight = FontWeight.W400,
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun NotesCardPreview() {
+    TrialFinishedNotesCard()
+}
+
+internal enum class TrialFinishedNotesItem(
+    override val icon: Int,
+    override val text: Int,
+) : NotesItem {
+    LOCK_FEATURE(
+        text = LR.string.plus_trial_finished_locked,
+        icon = PR.drawable.ic_locked_large,
+    ),
+    FILES_REMOVED(
+        text = LR.string.plus_trial_finished_files_removed,
+        icon = IR.drawable.ic_cloud,
+    ),
+    UPGRADE_ACCOUNT(
+        text = LR.string.plus_trial_finished_continue,
+        icon = IR.drawable.ic_plus,
+    ),
+}
+
+interface NotesItem {
```

Comment:
What's the purpose of this interface?

Reply:
Nice catch. It's not necessary anymore. removed